frontend/device/remarkable/device.lua

```diff
@@ -58,12 +72,28 @@ local wacom_width = 15725 -- unscaled_size_check: ignore
 local wacom_height = 20967 -- unscaled_size_check: ignore
 local wacom_scale_x = screen_width / wacom_width
 local wacom_scale_y = screen_height / wacom_height
-local TimeVal = require('ui/timeval')
-
-local adjustAbsEvt = function(self, ev)
-    -- for the rm2
-    ev.time = TimeVal:now()
+local mt_width = isRM2 and 1403 or 767 -- unscaled_size_check: ignore
+local mt_height = isRM2 and 1871 or 1023 -- unscaled_size_check: ignore
+local mt_scale_x = screen_width / mt_width
+local mt_scale_y = screen_height / mt_height
+
+local adjustTouchEvt = function(self, ev)
+    if isRM2 then
```

Comment:
I'm trusting upvalues here, but I'm pretty sure it will work fine.

ie: it worked on more f***d scenarios, like https://github.com/koreader/koreader/blob/master/frontend/device/android/device.lua#L51-L69, where we pass an android specific function to a generic handler.